src/daft-session/src/session.rs

```diff
@@ -263,6 +299,17 @@ impl SessionState {
             _ => panic!("ambiguous table identifier"),
         }
     }
+
+    pub fn get_function(&self, name: &str) -> Result<WrappedUDFClass> {
+        let mut items = self
+            .functions
+            .lookup(name, LookupMode::Insensitive)
+            .into_iter();
+        let f = items.next();
```

Comment:
need to error on ambiguous, but I guess that could be a todo. I sent you a DM about this, but perhaps we can discuss here since it's interesting to think where this session-specific error would go.

What are your thoughts on our crate-specific errors vs using errors in a common package? Right now daft-session is using the Error type from daft-catalog to avoid a lot of duplication, but now I want a session-specific error (ambiguous identifier error) but felt it was bad practice to add this to daft-catalog. I recall you had a comment about this before and would like your opinion here so I can start doing things better